etc/kayobe/kolla.yml

```diff
@@ -339,7 +339,8 @@ kolla_build_blocks:
     RUN sed -i 's/"pc-q35-rhel8.5.0"/"pc-q35-*"/' /usr/share/qemu/firmware/50-edk2-ovmf-cc.json
     {% endif %}
     {% endraw %}
-
+  magnum_base_footer: |
+    RUN curl -fsL https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | head -n -1 | bash
```

Comment:
Kolla base image already has curlrc which has relevant flags in, no need to pass them here again.